agent/src/main/java/com/teamscale/jacoco/agent/configuration/ProcessInformationRetriever.java

```diff
@@ -0,0 +1,66 @@
+package com.teamscale.jacoco.agent.configuration;
+
+import com.teamscale.client.ProcessInformation;
+import com.teamscale.report.util.ILogger;
+
+import java.lang.management.ManagementFactory;
+import java.lang.reflect.InvocationTargetException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+/**
+ * The ProcessInformationRetriever class is responsible for retrieving process information such as the host name and
```

Comment:
Consider following the JavaDoc guideline, in particular `Class/interface/field descriptions can omit the subject and simply state the object.`.